stdlib/2/inspect.pyi

```diff
@@ -81,9 +87,9 @@ Traceback = NamedTuple(
 
 _FrameRecord = Tuple[FrameType, str, int, str, List[str], int]
```

Comment:
I'd also rename this to `_FrameInfo` since in Python 3 it's a named tuple definition named `FrameInfo`.